charts/substra-backend/templates/_helpers.tpl

```diff
@@ -140,6 +140,18 @@ Return the user list
 {{- end }}
 {{- end -}}
 
+
+{{/*
+    Create the name fo the service account to use for the worker
+*/}}
+{{- define "substra.worker.serviceAccountName" -}}
+{{- if .Values.worker.serviceAccount.create -}}
+    {{ default (printf "%s-event" ( include "substra.fullname" .)) .Values.worker.serviceAccount.name }}
+{{- else -}}
+    {{ default "default" .Values.worker.serviceAccount.name }}
```

Comment:
the default value is "default"? I don't know what are best practices in that case but it looks like it could be improved

Reply:
Very good point, I didn't think to check one could fail out of helm templates with the `fail` template until your comment.